sdk/videoanalyzer/Azure.Media.VideoAnalyzer.Edge/src/autorest.md

```diff
@@ -0,0 +1,24 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+```yaml
+directive:
+- from: swagger-document
+  where: $.definitions.*
+  transform: >
+    $["x-csharp-usage"] = "model,input,output";
+    $["x-csharp-formats"] = "json";
+
+require: https://github.com/Azure/azure-rest-api-specs/blob/55b3e2d075398ec62f9322829494ff6a4323e299/specification/videoanalyzer/data-plane/readme.md
+azure-arm: false
+payload-flattening-threshold: 2
+license-header: MICROSOFT_MIT_NO_VERSION
+namespace: Microsoft.Azure.Media.VideoAnalyzer.Edge
+output-folder: $(csharp-sdks-folder)/videoanalyzer/Microsoft.Azure.Media.VideoAnalyzer.Edge/src/Generated
```

Comment:
This output folder also isn't correct. Shouldn't include "microsoft.". Seems to be ignored, though. Still, I'd just remove this. See sdk/search/Azure.Search.Documents/src/autorest.md for an example (we also have many transforms for an older swagger, but you can ignore those. Notice how few declarations we have, though. Ideally, most should be in the service's readme.md (or readme.csharp.md if C#-specific) so that all languages use the same settings mostly.